bundles/EcommerceFrameworkBundle/Model/AbstractOrder.php

```diff
@@ -201,19 +200,19 @@ public function getGiftItems()
     }
 
     /**
+     * @throws UnsupportedException
+     *
      * @param AbstractOrderItem[] $giftItems
      */
     public function setGiftItems($giftItems)
     {
-        // @TODO Throw UnsupportedException or change to abstract method in v7.0
-        Logger::err('setGiftItems not implemented for ' . get_class($this));
+        throw new UnsupportedException('setGiftItems not implemented for ' . get_class($this));
```

Comment:
Again, I'd revert this change for this PR.

Reply:
https://github.com/pimcore/pimcore/pull/7553